### PR TITLE
Fix(web)/dispute overview component tweaks

### PR DESCRIPTION
**Pull request by Harman-singh-waraich**
Warning ⚠️  : Requires subgraph redeploy

This PR fixes :

1.  Removes hardcoded Alias UI from Verdict component.
2. Dispute Timeline now shows on evidence period.
3.  Dispute Timeline dates have been fixed, we now have a `timeline` field in `Round` entity , that maps the Period ends

---

### Detailed summary

This PR focuses on adding the `createdAt` field to the `Round` entity in the GraphQL schema and updating the corresponding code to handle the new field. It also removes unused code related to user information and updates the timeline display in the `DisputeTimeline` component. Notable changes include:

- Added `createdAt` field to the `Round` entity in the GraphQL schema.
- Updated the `useVotingHistory` hook to include the `createdAt` field in the query.
- Updated the `createRoundFromRoundInfo` function to set the `createdAt` field for new rounds.
- Updated the `updateRoundTimeline` function to handle the `createdAt` field update.
- Removed unused code related to user information in the `Verdict` component.
- Updated the timeline display in the `DisputeTimeline` component to use formatted dates.

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: web/src/components/Verdict/DisputeTimeline.tsx [hunk 1]
```diff
@@ -75,19 +67,25 @@ const useItems = (disputeDetails?: DisputeDetailsQuery, arbitrable?: `0x${string
...
+          console.log(previousRoundEnd, index, roundTimeline);
+
           const icon = dispute.ruled && !rulingOverride && index === localRounds.length - 1 ? ClosedCaseIcon : "";
           const answers = disputeTemplate?.answers;
           acc.push({
             title: `Jury Decision - Round ${index + 1}`,
             party: isOngoing ? "Voting is ongoing" : getVoteChoice(parsedRoundChoice, answers),
-            subtitle: `${eventDate} / ${votingHistory?.dispute?.rounds.at(index)?.court.name}`,
+            //previous rounds endtime (appeal end) or dispute creation time if round 1 , if not ongoing the voteEnd time
+            subtitle: `${formatDate(isOngoing ? previousRoundEnd : roundTimeline?.[Periods.vote])} / ${
```
[the remark this answers is not in the document]
Comment (Harman-singh-waraich) [reply web/src/components/Verdict/DisputeTimeline.tsx:86]: yea makes sense 🤔 , will make like that

**Review by alcercu — APPROVED** [review]
lgtm

**Review by alcercu — CHANGES_REQUESTED** [review]
We are going to need to up the version of `subgraph/package.json` so it doesn't collide with the previous version when deploying

**Review by alcercu — DISMISSED** [review]
lgtm

**Pull request merged**